src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6134,6 +6134,9 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_TypeMustBePublic" xml:space="preserve">
     <value>Type '{0}' must be public to be used as a calling convention.</value>
   </data>
+  <data name="WRN_SyncAndAsyncEntryPoints" xml:space="preserve">
```

Comment:
>WRN_SyncAndAsyncEntryPoints [](start = 14, length = 27)

I'm not sure why CI didn't catch this, but adding a new warning requires a corresponding `_Title` suffixed entry. `TestDescriptorForConfigurableCompilerDiagnostics` should catch that.

Reply:
Did you run `eng\generate-compiler-code.cmd`? (CI correctness leg should theoretically catch it if you didn't...)

Reply:
Never mind. Saw the discussion in `GetSupportedErrorCodes`.